### PR TITLE
feat(brokerage): new checkout page for buy with cards has been implemented

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
This PR make our checkout for cards nicer :)

## Testing Steps (optional)
Here are screens of checkout 

image

image

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CheckoutConfirm/TooltipRowItem/index.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            <TextGroup inline>
+              <Text size='14px'>
+                <FormattedMessage
+                  id='modals.simplebuy.paying_with_card'
+                  defaultMessage='Blockchain.com requires a fee when paying with a card.'
+                />
+              </Text>
+              {/* TODO: update link */}
+              <Link
+                href='https://blockchain.zendesk.com/hc/en-us/sections/360002593291-Setting-Up-Lockbox'
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CheckoutConfirm/TooltipRowItem/index.tsx:93]: do we have a link yet? if not, can we just set it to `blockchain.com` so we will probably forget to update

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CheckoutConfirm/template.success.tsx [hunk 2]
```diff
@@ -123,61 +213,180 @@ const Success: React.FC<InjectedFormProps<{}, Props> & Props> = props => {
...
+                <TextGroup inline>
+                  <Text size='14px'>
+                    <FormattedMessage
+                      id='modals.simplebuy.paying_with_card'
+                      defaultMessage='Blockchain.com requires a fee when paying with a card.'
+                    />
+                  </Text>
+                  {/* TODO: update link */}
+                  <Link
+                    href='https://blockchain.zendesk.com/hc/en-us/sections/360002593291-Setting-Up-Lockbox'
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CheckoutConfirm/template.success.tsx:273]: same here

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CheckoutConfirm/template.success.tsx [hunk 3]
```diff
@@ -123,61 +213,180 @@ const Success: React.FC<InjectedFormProps<{}, Props> & Props> = props => {
...
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CheckoutConfirm/template.success.tsx:368]: same here

**Review by schnogz — CHANGES_REQUESTED** [review]

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CheckoutConfirm/template.success.tsx (hunk 3 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CheckoutConfirm/template.success.tsx:368]: we still do not have it :(

**Comment by milan-bc** [comment]
@schnogz will check with James, he was about to provide new pages

**Pull request merged**